src/cli/java/org/commcare/util/screen/EntityScreenContext.java

```diff
@@ -0,0 +1,70 @@
+package org.commcare.util.screen;
+
+/**
+ * Holds essential meta data associated with the entity screen
+ */
+public class EntityScreenContext {
+    private final int mOffSet;
+    private final String mSearchText;
+    private final int mSortIndex;
+    private final int mCasesPerPage;
+    private final String[] mSelectedValues;
```

Comment:
nit: maybe personal preference but I don't see the point of the `m` prefix, espeically since we're very inconsistent with it (even here `fuzzySearch` doesn't use it).

Reply:
Yeah I think it's a style preference. I just don't like having to use `this.var` in the constructor as it's unnecessary verbose.